### PR TITLE
Include projects and subprojects in the package path

### PR DESCRIPTION
**Pull request by ccamacho**
This commit includes the projects and subprojects folders as part of the topsail package path.

### CODE REVIEW

**Comment by kpouget** [comment]
@ccamacho ,
> topsail/subprojects/llm-load-test
this one actually belongs to `topsail/llm-load-test/subprojects/llm-load-test`
but when I did the relocation, the CI was failing to build the image because of it

so 1/ we need to make sure that you're not hitting the same issue
2/ if you're not hitting this issue, we can try to relocate it, there should be only one variable to change to update the location
(but right now, I can't remember where the location is specified :#)

File: requirements.txt [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+ansible
+jinja2
 flake8
 pylint
 awscli
 numpy
 joblib
 
 jsonpath_ng
-state-signals==0.5.2
```
Comment (ccamacho) [inline requirements.txt:13]: @kpouget quick question, is there a need for the pins on those packages versions? ansible, jinja2, and state-signals?

Comment (kpouget) [reply requirements.txt:13]: as long as nothing breaks, not really

File: .github/workflows/ansible-lint.yml [hunk 2]
```diff
@@ -22,7 +22,14 @@ jobs:
     - uses: actions/checkout@v3
 
     - name: Set ANSIBLE_ROLES_PATH
-      run: echo "ANSIBLE_ROLES_PATH=\"$(find ./projects -type d -name roles -print | tr '\n' ':')$(ls roles/* -d | tr '\n' ':')\"" >> $GITHUB_ENV
```
Comment (ccamacho) [inline .github/workflows/ansible-lint.yml:25]: There is no need to get each role path, we only need the 'roles' folder where they all are, and this path needs to be absolute. Also we should preserve the previous paths.

**Comment by kpouget** [comment]
@ccamacho , the test failed because Prow couldn't build the image, because it couldn't find something around git. Same issue I mentioned above :/ 

also, you can't launch the notebook and kserve test simultaneously (for OCP CI, it's the same test 🙃)

**Comment by kpouget** [comment]
closing as outdated

**Pull request closed without merging**